securedrop_client/logic.py

```diff
@@ -361,6 +361,7 @@ def login(self, username, password, totp):
         self.call_api(self.api.authenticate,
                       self.on_authenticate_success,
                       self.on_authenticate_failure)
+        self.update_sync()
```

Comment:
this can be removed-- we only want to get and show the latest last sync time:

* immediately on logout or logging in in offline mode or when the queue is paused
* every thirty seconds while offline

Reply:
it can be replaced with:

`self.set_status('')`

Reply:
Hmm... yes, `set_status` would work, but I like that `update_sync` should just do the "right thing" :tm: whenever it's called. I.e. it figures out the state and acts accordingly -- the behaviour for this is completely enclosed into a single function call. Using `set_status` feels like this is a step outside this behaviour, if you see what I mean. Absolutely concede this is just a sort of code aesthetic thing here, and (as always) happy to do whatever's needed. Just wanted to explain my thinking.